test/Centeva.ObjectStorage.IntegrationTests/Providers/AzureFileShareObjectStorageTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Centeva.ObjectStorage.Azure.FileShare;
+
+namespace Centeva.ObjectStorage.IntegrationTests.Providers;
+
+public class AzureFileShareObjectStorageFixture : ObjectStorageFixture
+{
+    public AzureFileShareObjectStorageFixture() : base("/" + nameof(AzureFileShareObjectStorageTests) + "/")
+    {
+    }
+
+    public override IObjectStorage CreateStorage(TestSettings settings)
+    {
+        return new AzureFileShareObjectStorage(settings.AzureAccountName!, settings.AzureAccountKey!, settings.AzureStorageTestContainerName!);
+    }
+
+    public override void Cleanup()
+    {
+    }
+}
+
+public class AzureFileShareObjectStorageTests : CommonObjectStorageTests, IClassFixture<AzureFileShareObjectStorageFixture>
+{
+    private readonly AzureFileShareObjectStorageFixture _fixture;
+
+    public AzureFileShareObjectStorageTests(AzureFileShareObjectStorageFixture fixture) : base(fixture)
+    {
+        _fixture = fixture;
+    }
+
+    // Test that azureFileShare implements ISupportsMetadata
```

Comment:
These comments add no value, since the method name accurately describes what's happening here

Reply:
I had turned off the viewing of whitespace changes, so I didn't realize the csproj had been modified outside of change the Description to include "Blob".